src/Sylius/Bundle/ArchetypeBundle/DependencyInjection/SyliusArchetypeExtension.php

```diff
@@ -29,88 +30,67 @@ class SyliusArchetypeExtension extends AbstractResourceExtension
      */
     public function load(array $config, ContainerBuilder $container)
     {
-        $config = $this->configure(
-            $config,
-            new Configuration(),
-            $container,
-            self::CONFIGURE_LOADER | self::CONFIGURE_DATABASE | self::CONFIGURE_PARAMETERS | self::CONFIGURE_VALIDATORS | self::CONFIGURE_TRANSLATIONS | self::CONFIGURE_FORMS
-        );
+        $config = $this->processConfiguration(new Configuration(), $config);
```

Comment:
this method seems to be copy/paste severals times! Could we share some code and move it to the `AbstractResourceExtension`